routers/repo/issue.go

```diff
@@ -1541,3 +1542,42 @@ func ChangeCommentReaction(ctx *context.Context, form auth.ReactionForm) {
 		"html": html,
 	})
 }
+
+// UpdateIssuePriority updates an issue priority.
+func UpdateIssuePriority(ctx *context.Context, form auth.EditPriorityForm) {
```

Comment:
IMHO it would be better to use the API endpoint you already created instead of having a second http endpoint which basically does the same thing.

Reply:
I'm not sure if you are talking about API `Pin`, `Unpin` and `UpdateIssuePriority`.  I just want to separate API `pin` and `unpin` from API `update` at first. Because I think pin is a special operation and we may handle it in a special way. However API `update` turns out to be a redundant one. I'll delete it if it's really unnecessary.